plugin.audio.intergalacticfm/main.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Module: default
+# Author: Pander
+# Created on: 2019-03-04
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from json import load
+from sys import argv
+from os.path import isfile
+import requests
+
+from urllib import urlencode
+from urlparse import parse_qsl
+
+
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+from xbmc import log, LOGNOTICE, LOGERROR
+
+__addonid__ = "plugin.audio.intergalacticfm"
+base = xbmc.translatePath('special://home/addons/{}/resources/'.format(__addonid__))
+
+# Get the plugin url in plugin:// notation.
+_url = argv[0]
+# Get the plugin handle as an integer number.
+_handle = int(argv[1])
+
+_addon = xbmcaddon.Addon()
+
+
+radio = 'http://radio.intergalactic.fm:80/'
+
+streams = load(open(base + 'streams.json'))
```

Comment:
Needs to be scoped in the `def list_audios()` method

Reply:
done